### DISCUSSION TITLE
Data transport from V6 to V10.5 ?

### DISCUSSION THREAD

**Discussion by moelski** [pimcore/pimcore#13652] [Q&A / Support]
Hello @all, 

we actually use Pimcore V6.9 as our main system. But we want to update to V10. 
I managed to install a fresh new v10 on a Linux system. 

Can anyone give me some hints how to get the PIM data from the V6 to the V10 system? 
We only us the pim functions of pimcore und habe Objects (with class Templates) and assets. 
If there any useable way to get the data to V10? I don´t want to update the old V6 system to V10. 

Thx 
Dominik

**Comment by aweichler** [comment]
Why don't you want to update?
It is not very hard to do and you can keep your data.

**Comment by moelski** [comment]
We need to update the server as well. It´s a very old ubuntu. 
And maybe this could also be updated, but the final decision was done by my boss. 
And now I have to evaluate possibilities to get all our data to the new server.

**Comment by aweichler** [comment]
Maybe you can update pimcore to 10 localy and put it to an other server and re run the update process with fresh data. 
But maybe it would be better to start using docker on the new server. So updates for the host and for the container would be much easier.
